todoist.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+FLAGS=''
+
+if [[ $XDG_SESSION_TYPE == "wayland" ]] && [ -c /dev/nvidia0 ]
+then
+    FLAGS="$FLAGS --disable-gpu-sandbox"
+fi
+
+env TMPDIR="$XDG_RUNTIME_DIR/app/${FLATPAK_ID:-com.todoist.Todoist}" zypak-wrapper /app/extra/todoist/todoist $FLAGS "$@"
```

Comment:
Why change the TMPDIR? It already points to `$HOME/.var/app/com.todoist.Todoist/cache`, why change it?

Reply:
I asked the author or zypack and they wrote:
>  The first sets the tmpdir to an actual shared tmpfs, the second sets it to somewhere persistent, would recommend the first unless the program uses very large temporary files

But I'm happy to change it back if you want.

Reply:
Ah, gotcha. Seems like the the [documentation](https://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access) for this changed since I last worked with this. It use to just recommend using `/var/tmp`.

I would do it without the default value however, like in the docs. It should never happen that the id is not set so there is no need in hardcoding the id if its not needed.